recipes/cjson/all/conanfile.py

```diff
@@ -15,11 +15,13 @@ class CjsonConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_utils": [True, False],
```

Comment:
The _unofficial_ convention is to have `with_` but it's not a hard requirement!

Reply:
Technically, `with_` should be used for `fPIC` as well... but of course, I can change that. No need to violate *unofficial* conventions with the first try. ;-)